domain/service/latest_entry/latest_entry.go

```diff
@@ -0,0 +1,39 @@
+package latest_entry
```

Comment:
**[golint]** <sub>reported by [reviewdog](https://github.com/haya14busa/reviewdog) :dog:</sub>
don't use an underscore in package name